14_functions_lab/04.py

```diff
@@ -1,6 +1,8 @@
-"""
-Write a function that takes minlen and
-a list of words, and returns only the words
-longer than minlen
-"""
-
+def longer_than(minLength, *list):
+    refinedList = []
+    for word in list:

```

Comment:
Notice how github colours "list" in the same color as print (and differently from "refinedList" and "word"). The reason is python has a function named list, so when you use that as a variable name it can be a bit confusing.